stdlib/public/core/Codable.swift

```diff
@@ -3845,237 +3843,237 @@ internal class _KeyedDecodingContainerBase<Key: CodingKey> {
     fatalError("_KeyedDecodingContainerBase cannot be used directly.")
   }
 
-  internal var allKeys: [Key] {
+  internal var allKeys: [CodingKey] {
```

Comment:
@jckarter I don't suppose you have any ideas for improving this one?

Reply:
Hm. Do we have a way using standard library SPI that we could just store the reference to the array's buffer object here, and construct the `Array<Key>` shell around it in the getter?

Reply:
If the SPI doesn't exist, maybe you could have this variable be of type `_ArrayBridgeStorage` and bitcast it back and forth to `Array<K>` in the methods that access it.